.github/workflows/e2e-test.yml

```diff
@@ -44,10 +44,10 @@ jobs:
         exclude:
           - test_groups: 'blocks'
             test_branches: 'merchant'
-          - woocommerce: '5.2.0'
+          - woocommerce: '5.4.0'
             test_groups: 'blocks'
             test_branches: 'shopper'
-          - woocommerce: '6.1.0'
+          - woocommerce: '6.5.1'
```

Comment:
This does not sound correct to me. @achyuthajoy and @deepakpathania - can you have a look to confirm this change? 

Also, I think it's worth commenting why we exclude these combinations?

Reply:
I wasn't sure why it was excluded in the previous release, so I used the same pattern.

Reply:
@htdat @zmaglica These test groups were excluded because the minimum supported WC version for WC Blocks. If we're running Blocks tests against an unsupported WC version, the tests will fail and hence they're excluded from testing.

At the moment, the minimum supported WC version for Blocks is `6.3` - https://github.com/woocommerce/woocommerce-blocks/blob/07a0b46a568e6187981cfd89467c78600c82f2af/woocommerce-gutenberg-products-block.php#L12

Reply:
@zmaglica You can remove exclusion for WC versions higher than `6.3`. So in this case, keep exclusion for `5.4.0` which is lower than the minimum supported WC version for blocks.

Reply:
Here is the PR that adds this change - https://github.com/Automattic/woocommerce-payments/pull/4172